third_party/move/move-compiler-v2/tests/bytecode-generator/fields.exp

```diff
@@ -15,14 +15,32 @@ module 0x42::fields {
     }
     private fun write_local_direct(): fields::S {
         {
-          let x: fields::S = pack fields::S(0, pack fields::T(0));
+          let x: fields::S = {
+            let .f: u64 = 0;
```

Comment:
[citation needed]
Is this documented anywhere?  If not, @wrwg please do so.

Reply:
> As said in the last round, our convention is to start temps with '$'. This is important to identify because those names should not be shown in error messages. To avoid clash with other '$' vars, you could generated `$S_f`.

Just realized that shadowing is actually not a problem here:
```
let $f = 0;
pack { f: $f + 1 }
```
will still produce
```
let $f = 0;
{
    let $f = $f + 1;
    pack {f : $f}
}
```
correctly.